### ISSUE TITLE
Not enough information in documentation to start development

### ISSUE THREAD

**Issue by sreenathkumar** [RocketChat/EmbeddedChat#210]
### After cloning the repo I run `npm i` then this errors has come. I can't understand what is it and how to fix.

![Screenshot_150](https://user-images.githubusercontent.com/60534594/230404986-84f8b2bc-b602-46df-aabd-1238f74b20d8.png)
![Screenshot_153](https://user-images.githubusercontent.com/60534594/230405129-51dbfddf-6332-43da-86e6-377993eafc65.png)
![Screenshot_152](https://user-images.githubusercontent.com/60534594/230405143-64e15e48-4437-41a3-87be-2a96e28beab0.png)
![Screenshot_151](https://user-images.githubusercontent.com/60534594/230405145-573b6292-2d4a-431c-9827-cf52e38d4b36.png)
![Screenshot_154](https://user-images.githubusercontent.com/60534594/230405149-959a5108-5923-4c1b-afcd-0006df877700.png)

### Here is the two log files I found there:
[2023-04-06T14_07_33_228Z-debug.log](https://github.com/RocketChat/EmbeddedChat/files/11171503/2023-04-06T14_07_33_228Z-debug.log)
[eresolve-report.txt](https://github.com/RocketChat/EmbeddedChat/files/11171504/eresolve-report.txt)

### When I run `npm i` in the playground folder this error come:
![Screenshot_155](https://user-images.githubusercontent.com/60534594/230406137-9372790b-41ce-4cdf-a670-02a888ee9240.png)

### Ultimately when I run` npm run dev`
![image](https://user-images.githubusercontent.com/60534594/230406735-3067b2ec-6f11-485c-b81f-8ed57364cad9.png)

Basically my intension is to contribute in this project. So i think you will give me a solution to this issues. 

Thanks in advance
-- Sreenath

**Comment by sidmohanty11** [comment]
Hey @sreenathkumar , sorry for the late reply. Yup, as of now EC builds are failing due to broken dependency tree but you can still try it out by doing,
```
npm i --legacy-peer
cd playground
npm i
cd ..
npm run dev
```

Let me know if it works : )

**Comment by sreenathkumar** [comment]
@sidmohanty11 Now it's stucked here.

![image](https://github.com/RocketChat/EmbeddedChat/assets/60534594/9c6d4153-2ee7-4db8-b12d-6940f87cd495)

**Comment by sidmohanty11** [comment]
I see @sreenathkumar , you can see in the logs that PORT is not recognized. This is because the commands are specific to linux based terminals.

You can try this: https://stackoverflow.com/questions/47806623/unrecognized-port-command

Also, this is an amazing catch, it would be awesome if we can make a script to handle all this. What do you think?

**Comment by umangutkarsh** [comment]
Hi @sreenathkumar , has this issue been resolved?

**Comment by sreenathkumar** [comment]
@umangutkarsh no

**Comment by Spiral-Memory** [comment]
Closing due to inactivity.
